components/patient-data/rest/src/main/java/org/phenotips/data/rest/internal/DefaultPatientsSuggestionsResourceImpl.java

```diff
@@ -239,14 +233,6 @@ private void appentPatientXML(Patient patient, StringBuilder xmlResult, boolean
 
         String description = getDescription(patient);
 
-        // Add family info
-        Family family = this.familyRepository.getFamilyForPatient(patient);
```

Comment:
We do not include family in XML response?

Reply:
Hm, looks like I forgot to include something in the commit...

Reply:
Ah no, I remembered I did it intentionally. The suggest doesn't care about extra attributes, and having the family in the label isn't required but would make the code messier.